instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo/test/doc.go

```diff
@@ -0,0 +1,11 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+Package test validates the otelmongo V2 instrumentation with the default SDK.
+
+This package is in a separate module from the instrumentation it tests to
```

Comment:
Having this in a test subpackage shouldn't be required anymore, and we're in the process of removing that folder for every instrumentation.
Since this is a new one, could we avoid creating it altogether?

See https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6763 as similar work for otelhttp.

Reply:
sure.  I can move the tests to the package folder similar to what was done in https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6763